scripts/install-vsix-dependencies.js

```diff
@@ -28,27 +28,9 @@ const darwinExecutable = path.join(
 const linuxExecutable = path.join(
   testRunFolderAbsolute,
   'VSCode-linux-x64',
+  'bin',
   'code'
```

Comment:
This is the actual location the executable that can understand `code --install-extension....`